TESTING.md

```diff
@@ -532,6 +533,33 @@ Look at `TestResourceInfoProviders` to see a full example of how to use the meth
 once in the HCL script. If it is not, the variable `testAccProvider` may not get initialised, and if that happens,
 test checks that use the expression `conn := testAccProvider.Meta().(*VCDClient)` will panic.
 
+## Partitioned tests
+
+We can run tests in a partitioned way, by splitting the tests across several VCDs (nodes) each of which will run a given portion of the tests.
+
+To activate this modality, we use the following options:
+
+* `-vcd-partitions=N` indicates the number of partitions running the test suite.
+* `-vcd-partition-node=N` indicates which node the current VCD will run.
+* `-vcd-partition-tests-file=FileName` (optional) provides the list of tests to run
+
+When partition mode is enabled, the current node will skip all the tests that are not assigned to it.
+
+The test assignment, by default, happens by getting the list of all tests, sorting them alphabetically, and then assigning them to each node in sequence. 
+If a tests file name was provided, the node will just use such file as its assignment.
+
+Each node produces several files:
+
+* `{BUILD_NUMBER}-tests-retrieved-in-node-{NODE_NUMBER}-{VCD_VERSION}.txt` All the tests found in the current node
```

Comment:
While `NODE_NUMBER` and `VCD_VERSION` meaning is trivial (imho), I think that `BUILD_NUMBER` maybe deserves a comment. Like, it is just "an ID" of the test run, or that it's optional (its value will be `LOCAL` if not set).

Reply:
Docs updated